Makefile

```diff
@@ -132,6 +132,10 @@ build/addon/data/panel-bundle.js: $(shoot_panel_dependencies)
 	@mkdir -p $(@D)
 	# Save the bundle dependencies:
 	browserify --list -e ./build/addon/data/shoot-panel.js | sed "s!$(shell pwd)/!!g" > build/shoot-panel-dependencies.txt
+	cat ./build/addon/lib/shooter.js | ./bin/_fixup_panel_js > ./build/addon/lib/shooter-new.js
+	mv ./build/addon/lib/shooter-new.js ./build/addon/lib/shooter.js
```

Comment:
addon/lib/shooter.js isn't, it's an addon module.

You are correct about shoot-panel.js – another application of -prune should fix that one.